R/Reactant/build_tarballs.jl

```diff
@@ -7,15 +7,18 @@ include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
 
 name = "Reactant"
 repo = "https://github.com/EnzymeAD/Reactant.jl.git"
-version = v"0.0.15"
+version = v"0.0.16"
```

Comment:
I'm releasing a new build in #9321 with the new API functions so this should be v0.0.17

```suggestion
version = v"0.0.17"
```

Reply:
er, because of the bazel requirement update [which this PR is trying to work around, this is going to be needed first].